src/ol_dbt/models/staging/ocw/_stg_ocw__models.yml

```
@@ -0,0 +1,220 @@
+---
+version: 2
+
+models:
+- name: stg__ocw__studio__postgres__websites_website
+  description: OCW websites metadata on production environment. A website can refers
+    to OCW courses, OCW website configuration, etc.
+  columns:
+  - name: website_uuid
+    description: str, auto-generated unique identifier string to identify the OCW
+      website
+    tests:
+    - not_null
+    - unique
+  - name: website_name
+    description: str, unique name of the website. It appears as part of website url
```

```suggestion
  - name: website_name
    description: str, unique name of the website. It appears as part of OCW Studio urls
```

@pt2302 can you confirm my understanding here? I believe the website_name is only used in internal, Studio URLs.